spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -456,7 +456,7 @@ def setup_gui(self):
         scrollarea = self.setup_scrollarea()
         up_btn, down_btn = self.setup_arrow_buttons()
 
-        self.setFixedWidth(135)
+        self.setFixedWidth(150)
```

Comment:
I would like not to hardcode a value here anymore. I will come back with a solution so that you don't waste time on this.

Reply:
Off course, I have no problem with that, please go ahead.

Reply:
Ok, thanks!